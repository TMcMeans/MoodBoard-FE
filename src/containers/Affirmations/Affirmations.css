.month-affirmations {
  height: 100vh;
}

.month {
  width: 300px;
  border-bottom: 1px solid #fff;
  margin: 0 auto;
  color: #fff;
}

.affirmation-list {
  height: 55%;
  padding: 0;
}

.single-affirmation {
  color: #fff;
  display: flex;
  flex-direction: row;
  font-size: 1.5em;
  justify-content: center;
  margin: 1rem;
}

.circle {
  color: #4BCE8F;
}

.affirmation-text {
  margin-left: 1.5rem;
}

.mood-color {
  width: 20px;
  height: 20px;
  border-radius: 20px;
  margin: auto 10px auto 0;
}
