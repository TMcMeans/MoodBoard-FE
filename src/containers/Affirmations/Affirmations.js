import React, { Component } from 'react';
import { FaCircle } from 'react-icons/fa';
import PropTypes from 'prop-types';

import Logo from '../../components/Logo/Logo';
import Button from '../../components/Button/Button';

import './Affirmations.css';

export class Affirmations extends Component {
  constructor() {
    super();
    this.state = {};
  }

  render() {
    const monthText = 'June';
    return (
      <div className="month-affirmations">
        <Logo />
        <h1 className="month">{monthText} Affirmations</h1>
        <ul className="affirmation-list">
          <li className="single-affirmation" id="1">
            <FaCircle className="circle" />
            <span className="affirmation-text">6/3: "You got this."</span>
          </li>
          <li className="single-affirmation" id="2">
            <FaCircle className="circle" />
            <span className="affirmation-text">6/3: "You got this."</span>
          </li>
          <li className="single-affirmation" id="3">
            <FaCircle className="circle" />
            <span className="affirmation-text">6/3: "You got this."</span>
          </li>
          <li className="single-affirmation" id="4">
            <FaCircle className="circle" />
            <span className="affirmation-text">6/3: "You got this."</span>
          </li>
        </ul>
        <button className="standard-btn light back-to-calendar-btn">
          calendar
        </button>
      </div>
    );
  }
}

export default Affirmations;
