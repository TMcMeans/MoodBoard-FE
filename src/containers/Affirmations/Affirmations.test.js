import React from 'react';
import { shallow } from 'enzyme';
import { Affirmations } from './Affirmations';

describe('Affirmations', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Affirmations />);
  });
  it('should render a snapshot with correct data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
