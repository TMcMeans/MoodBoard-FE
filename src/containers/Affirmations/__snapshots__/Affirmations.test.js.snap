// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Affirmations should render a snapshot with correct data passed in 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Affirmations />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Logo />,
        <h1
          className="month"
        >
          Febuary
           Affirmations
        </h1>,
        <ul
          className="affirmation-list"
        />,
        <button
          className="standard-btn light back-to-calendar-btn"
        >
          calendar
        </button>,
      ],
      "className": "month-affirmations",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Febuary",
            " Affirmations",
          ],
          "className": "month",
        },
        "ref": null,
        "rendered": Array [
          "Febuary",
          " Affirmations",
        ],
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [],
          "className": "affirmation-list",
        },
        "ref": null,
        "rendered": Array [],
        "type": "ul",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "calendar",
          "className": "standard-btn light back-to-calendar-btn",
        },
        "ref": null,
        "rendered": "calendar",
        "type": "button",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Logo />,
          <h1
            className="month"
          >
            Febuary
             Affirmations
          </h1>,
          <ul
            className="affirmation-list"
          />,
          <button
            className="standard-btn light back-to-calendar-btn"
          >
            calendar
          </button>,
        ],
        "className": "month-affirmations",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Febuary",
              " Affirmations",
            ],
            "className": "month",
          },
          "ref": null,
          "rendered": Array [
            "Febuary",
            " Affirmations",
          ],
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [],
            "className": "affirmation-list",
          },
          "ref": null,
          "rendered": Array [],
          "type": "ul",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "calendar",
            "className": "standard-btn light back-to-calendar-btn",
          },
          "ref": null,
          "rendered": "calendar",
          "type": "button",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
